appveyor.yml

```diff
@@ -12,12 +12,12 @@ clone_folder: C:\gopath\src\github.com\influxdata\telegraf
 environment:
   GOPATH: C:\gopath
 
-stack: go 1.14
+stack: go 1.15
 
 platform: x64
 
 install:
-  - choco install make
+  - choco install make mingw
```

Comment:
What are we using mingw to do?  It's a relatively big package that includes gcc.  Are there tests that depend on it?

Reply:
Right, it is not needed (`gcc` from it was probably needed for some test which was disabled)